### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by hbakhtiyor**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -149,6 +149,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [Facebook](https://developers.facebook.com/docs/plugins) - Facebook embedded plugins.
 - [ShareThis](http://www.sharethis.com/) - Sharing buttons for multiple social networks.
 - [Storify](https://storify.com/) - Embed various social media in one place.
+- [Kontaktify](https://www.kontaktify.com/) - A perfect contact widget for your website.
```
Comment (waldyrious) [inline readme.md]: I don't think "perfect" is useful here -- perhaps some other adjective (an objective one, preferably) could work better. The "your website" bit is also unnecessary.

**Comment by agarrharr** [comment]
@waldyrious I agree.

@hbakhtiyor Do you want to change the wording?

**Comment by hbakhtiyor** [comment]
np, can you update it?

**Comment by hbakhtiyor** [comment]
@agarrharr thanks

**Pull request merged**